astropy/io/votable/tests/test_table.py

```diff
@@ -52,6 +52,49 @@ def home_is_tmpdir(monkeypatch, tmp_path):
     monkeypatch.setenv("USERPROFILE", str(tmp_path))
 
 
+def test_read_write_votable_parquet(tmp_path):
+    """
+    Test to write VOTable with Parquet serialization
+    """
+
+    # Create some fake data
+    number_of_objects = 10
+    ids = [f"COSMOS_{ii:03g}" for ii in range(number_of_objects)]
+    redshift = np.random.uniform(low=0, high=3, size=number_of_objects)
+    mass = np.random.uniform(low=1e8, high=1e10, size=number_of_objects)
+    sfr = np.random.uniform(low=1, high=100, size=number_of_objects)
+    astropytab = Table([ids, redshift, mass, sfr], names=["id", "z", "mass", "sfr"])
+
+    # Create Column metadata
+    column_metadata = {
+        "id": {"unit": "", "ucd": "meta.id", "utype": "none"},
+        "z": {"unit": "", "ucd": "src.redshift", "utype": "none"},
+        "mass": {"unit": "solMass", "ucd": "phys.mass", "utype": "none"},
+        "sfr": {"unit": "solMass yr-1", "ucd": "phys.SFR", "utype": "none"},
+    }
+
+    # Write VOTable with Parquet serialization
+    filename = os.path.join(tmp_path, "test_votable_parquet.vot")
+    astropytab.write(
+        filename=filename,
+        column_metadata=column_metadata,
+        overwrite=True,
+        format="votable.parquet",
+    )
+
+    # Open created VOTable with Parquet serialization
+    votable = parse(filename)
```

Comment:
The table write is done with the Table I/O registry but the read is done with the lower-level `parse` instead.

Reply:
the `parse` parses the VOTable correctly and creates a VOTable object. Reading the tables only also works with Table.read(filename). It adds the units correctly, but does not seem to have the other quantities as a VOTable object. 
What would you suggest?

Reply:
I see, it is necessary to read the table back in as a VOTable to have access to the metadata. That justifies leaving the use of `parse` as-is.